rollup/plugins/bundlePlugins.js

```diff
@@ -164,6 +167,37 @@ export function makeTSPlugin(jsVersion) {
   return plugin;
 }
 
+/**
+ * Creates a Rollup plugin that removes all code between the `__ROLLUP_EXCLUDE_FROM_BUNDLES_BEGIN__`
+ * and `__ROLLUP_EXCLUDE_FROM_BUNDLES_END__` comment guards. This is used to exclude the Replay integration
+ * from the browser and browser+tracing bundles.
+ * If we need to add more such guards in the future, we might want to refactor this into a more generic plugin.
```

Comment:
Very happy that we're pragmatic here for now!

Reply:
Yup, as I wrote, this surely can be made more general but let's worry about this when we need to.